pwiz_tools/Skyline/Model/SmallMoleculeTransitionListReader.cs

```diff
@@ -201,8 +201,14 @@ public SrmDocument CreateTargets(SrmDocument document, IdentityPath to, out Iden
         private bool ErrorAddingToExistingMoleculeGroup(ref SrmDocument document, ParsedIonInfo precursor, string groupName,
             string defaultPepGroupName, HashSet<MoleculeNameAndAccessions> moleculeIDsSeen, Row row, ref bool pepGroupFound)
         {
-            var adduct = precursor.Adduct;
+            var adduct = precursor?.Adduct;
+            if (Adduct.IsNullOrEmpty(adduct))
+            {
+                return true; // Can't process
+            }
+            // ReSharper disable PossibleNullReferenceException
             var precursorMonoMz = adduct.MzFromNeutralMass(precursor.MonoMass);
+            // ReSharper restore PossibleNullReferenceException

```

Comment:
An alternative way to suppress the ReSharper warning would be to put an exclamation mark before the ".":
`adduct!.MzFromNeutralMass`